src/app/main.js

```diff
@@ -23,6 +23,8 @@ const GlobalDirective = require('gui/vue/vue.directives');
  */
 window.g3wsdk = require('api');
 
+require('app/dev');
```

Comment:
NB the `src/app/dev/inde.js` file is now included much earlier in the execution flow